### PR TITLE
Log graph

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

* Removes log graph section from the `logs/explore` page
* Creates a new doc page dedicated to log graph
 
### Motivation

Now we have more than one log graph to document

### Preview link

* https://docs-staging.datadoghq.com/gus/log-graph/logs/graph/

### CODE REVIEW

File: content/logs/graph.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    
+    {{< img src="logs/graph/choose_measure_facet.png" alt="choose measure facet" responsive="true" popup="true" style="width:50%;">}}
+2. Select the aggregation function for the [Measure][1] you want to graph:
+    
+    {{< img src="logs/graph/agg_function_log_graph.png" alt="aggregation function for log graph" responsive="true" popup="true" style="width:50%;">}}
+3. Split by [Tag][1] or [Facet][2] to split your graph over the desired dimension.
+
+    {{< img src="logs/graph/split_by_log_graph.png" alt="split by log graph" responsive="true" popup="true" style="width:50%;">}}
+
+4. Choose to display either the *X* **top** or **bottom** values according to the selected [measure][1]
```
Comment (phrawzty) [inline content/logs/graph.md]: missing period at the end

File: content/logs/graph.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    
+    {{< img src="logs/graph/top_bottom_button.png" alt="top bottom button" responsive="true" popup="true" style="width:20%;">}}
+
+## Visualizations 
+
+Choose which log graph visualization you want to use with the graph selector:
+
+{{< img src="logs/graph/graph_selector.png" alt="log graph selector" responsive="true" popup="true" style="width:30%;">}}
+
+Available visualization are:
```
Comment (phrawzty) [inline content/logs/graph.md]: > Available visualizations:

File: content/logs/graph.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+* [Timeseries](#timeseries)
+* [Top List](#top-list)
+
+### Timeseries
+
+See a single [measure][1] evolution over a selected time frame and optionally splited by an available [facet][2].
```
Comment (phrawzty) [inline content/logs/graph.md]: > Visualize the evolution of a single measure over a selected time frame, and (optionally) split by an available facet.

File: content/logs/graph.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+further_reading:
+- link: "logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+---
+
+Switch between the Log List and the Log Graph mode by clicking on this button:
```
Comment (phrawzty) [inline content/logs/graph.md]: Does this button have a name? If so, it should be used, such as:
> Switch between the Log List and the Log Graph modes by clicking on the _Log Mode_ button:

File: content/logs/graph.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+{{< img src="logs/graph/log_graph_switch.png" alt="Log graph switch" responsive="true" popup="true" style="width:40%;">}}
+
+## Log graph query 
+
+Use the query to control what's displayed in your log graph:
```
Comment (phrawzty) [inline content/logs/graph.md]: "Log Graph" is capitalised above, but lower-case here. Is that correct? (I don't know!)

File: content/logs/graph.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+1. Choose a [Measure][1] or [Facet][2] to graph. Choosing a [Measure][1] lets you choose the aggregation function whereas selecting a [Facet][2] displays the unique count.
```
Comment (phrawzty) [inline content/logs/graph.md]: > Choose a Measure or Facet to graph. Measure lets you choose the aggregation function whereas Facet displays the unique count.

File: content/logs/graph.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+    
+    {{< img src="logs/graph/choose_measure_facet.png" alt="choose measure facet" responsive="true" popup="true" style="width:50%;">}}
+2. Select the aggregation function for the [Measure][1] you want to graph:
+    
+    {{< img src="logs/graph/agg_function_log_graph.png" alt="aggregation function for log graph" responsive="true" popup="true" style="width:50%;">}}
+3. Split by [Tag][1] or [Facet][2] to split your graph over the desired dimension.
```
Comment (phrawzty) [inline content/logs/graph.md]: > Use Tag or Facet to split your graph […]

File: content/logs/graph.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+
+    {{< img src="logs/graph/split_by_log_graph.png" alt="split by log graph" responsive="true" popup="true" style="width:50%;">}}
+
+4. Choose to display either the *X* **top** or **bottom** values according to the selected [measure][1]    
+    
+    {{< img src="logs/graph/top_bottom_button.png" alt="top bottom button" responsive="true" popup="true" style="width:20%;">}}
+
+## Visualizations 
+
+Choose which log graph visualization you want to use with the graph selector:
```
Comment (phrawzty) [inline content/logs/graph.md]: > Select a log graph visualization type using the graph selector:

File: content/logs/graph.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+
+* [Timeseries](#timeseries)
+* [Top List](#top-list)
+
+### Timeseries
+
+See a single [measure][1] evolution over a selected time frame and optionally splited by an available [facet][2].
+
+The following log graph can be read as:  
+The evolution of the **top 5** of **Url Path** according to the amount of **Unique client IP**  for each one of them over the last month.
```
Comment (phrawzty) [inline content/logs/graph.md]: > The evolution of the top 5 URL Paths according to the number of unique Client IPs over the last month.

File: content/logs/graph.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+
+{{< img src="logs/graph/timeserie_example.png" alt="timeserie example" responsive="true" popup="true" style="width:90%;">}}
+
+### Top List 
+
+Consult the top according to a [measure][1] for a given [facet][2]:
+
+The following log graph can be read as:  
+The **top 5** of **Url Path** according to the amount of Unique **client IP**  for each one of them
```
Comment (phrawzty) [inline content/logs/graph.md]: > The evolution of the top 5 URL Paths according to the number of unique Client IPs over the last month.

File: content/logs/graph.md [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+Available visualization are:
+
+* [Timeseries](#timeseries)
+* [Top List](#top-list)
+
+### Timeseries
+
+See a single [measure][1] evolution over a selected time frame and optionally splited by an available [facet][2].
+
+The following log graph can be read as:
```
Comment (phrawzty) [inline content/logs/graph.md]: Can it be read any other way? _If no_, be concrete, as in:
> The following line graph shows:

File: content/logs/graph.md [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+The following log graph can be read as:  
+The evolution of the **top 5** of **Url Path** according to the amount of **Unique client IP**  for each one of them over the last month.
+
+{{< img src="logs/graph/timeserie_example.png" alt="timeserie example" responsive="true" popup="true" style="width:90%;">}}
+
+### Top List 
+
+Consult the top according to a [measure][1] for a given [facet][2]:
+
+The following log graph can be read as:
```
Comment (phrawzty) [inline content/logs/graph.md]: Can it be read any other way? _If no_, be concrete, as in:
> The following line graph shows:

File: content/logs/graph.md [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+See a single [measure][1] evolution over a selected time frame and optionally splited by an available [facet][2].
+
+The following log graph can be read as:  
+The evolution of the **top 5** of **Url Path** according to the amount of **Unique client IP**  for each one of them over the last month.
+
+{{< img src="logs/graph/timeserie_example.png" alt="timeserie example" responsive="true" popup="true" style="width:90%;">}}
+
+### Top List 
+
+Consult the top according to a [measure][1] for a given [facet][2]:
```
Comment (phrawzty) [inline content/logs/graph.md]: The top _what_?

File: content/logs/graph.md [hunk 14]
```diff
@@ -0,0 +1,90 @@
...
+
+Select or click on a section of the graph to either zoom in the graph or see the list of logs corresponding to your selection:
+
+{{< img src="logs/graph/view_logs.gif" alt="view logs" responsive="true" popup="true" style="width:80%;">}}
+
+## Export
+
+{{< img src="logs/graph/export_button.png" alt="view logs button" responsive="true" popup="true" style="width:40%;">}}
+
+Export your Log graph:
```
Comment (phrawzty) [inline content/logs/graph.md]: Capital L but lower-case g?

File: content/logs/graph.md [hunk 15]
```diff
@@ -0,0 +1,90 @@
...
+{{< img src="logs/graph/view_logs.gif" alt="view logs" responsive="true" popup="true" style="width:80%;">}}
+
+## Export
+
+{{< img src="logs/graph/export_button.png" alt="view logs button" responsive="true" popup="true" style="width:40%;">}}
+
+Export your Log graph: 
+
+* To create a new [log monitor][4]:  
+    The query of your log graph is used to create the log monitor query
```
Comment (phrawzty) [inline content/logs/graph.md]: add a period at the end

File: content/logs/graph.md [hunk 16]
```diff
@@ -0,0 +1,90 @@
...
+* To an existing [Timeboard][5]:  
+    This functionality is not released yet, stay tuned ;)
```
Comment (phrawzty) [inline content/logs/graph.md]: […] yet–stay tuned!

(Ugh Y dinnae lyke emdashes)

File: content/logs/graph.md (hunk 5 above)
Comment (l0k0ms) [reply content/logs/graph.md]: it should be capitalised indeed :)

File: content/logs/graph.md [hunk 17]
```diff
@@ -11,54 +11,54 @@ further_reading:
...
-The evolution of the **top 5** of **Url Path** according to the amount of **Unique client IP**  for each one of them over the last month.
+The following timeseries Log Graph shows:
+The evolution of the **top 5 URL Paths** according to the number of **unique Client IPs** over the last month.
 
 {{< img src="logs/graph/timeserie_example.png" alt="timeserie example" responsive="true" popup="true" style="width:90%;">}}
 
 ### Top List 
 
-Consult the top according to a [measure][1] for a given [facet][2]:
+Consult the top of values from a [facet][2] according to the chosen [measure][1]:
```
Comment (phrawzty) [inline content/logs/graph.md]: > Consult the top values from […]

`s/of//;`

File: content/logs/graph.md [hunk 18]
```diff
@@ -72,12 +72,12 @@ Select or click on a section of the graph to either zoom in the graph or see the
...
 
-Export your Log graph: 
+Export your Log Graph: 
 
 * To create a new [log monitor][4]:  
-    The query of your log graph is used to create the log monitor query
+    The query of your Log Graph is used to create the log monitor query.
 * To an existing [Timeboard][5]:  
-    This functionality is not released yet, stay tuned ;)
+    This functionality is not released yet–stay  tuned ;)
```
Comment (phrawzty) [inline content/logs/graph.md]: I don't think we do winky-faces in our documentation. Suggest removing it and terminating the phrase with a period or, if you're feeling saucy, an exclamation point.

File: content/logs/graph.md [hunk 19]
```diff
@@ -0,0 +1,90 @@
...
+{{< img src="logs/graph/graph_selector.png" alt="Log Graph selector" responsive="true" popup="true" style="width:30%;">}}
+
+Available visualizations:
+
+* [Timeseries](#timeseries)
+* [Top List](#top-list)
+
+### Timeseries
+
+Visualize the evolution of a single [measure][1] over a selected time frame, and (optionally) split by an available [facet][2].
```
Comment (NBParis) [inline content/logs/graph.md]: of a single measure `or facet unique count`

**Pull request merged**